### PR TITLE
Add registerNonModuleBeacons

### PR DESCRIPTION
**Pull request by FHieser**
In the registerNonModuleBeacons function should I check if the implementation of the beacon is not a module?

### CODE REVIEW

**Comment by FHieser** [comment]
@marvinkruse could you please review?

File: src/external/governance/Governor_v1.sol [hunk 1]
```diff
@@ -568,4 +579,14 @@ contract Governor_v1 is
         // we are here, that means target is owned by this address and inverter beacon
         return true;
     }
+
+    /// @dev	Internal function that adds a beacon to the linked beacons array.
+    /// @param  beacon the beacon to add.
```
Comment (marvinkruse) [inline src/external/governance/Governor_v1.sol]: Tiny capitalization issue 😇
```suggestion
    /// @param  beacon The beacon to add.
```

File: src/external/governance/interfaces/IGovernor_v1.sol [hunk 2]
```diff
@@ -267,6 +271,11 @@ interface IGovernor_v1 {
         IInverterBeacon_v1 beacon
     ) external;
 
+    /// @notice Registers a {IInverterBeacon_v1} as a linked beacon that is not a module.
```
Comment (marvinkruse) [inline src/external/governance/interfaces/IGovernor_v1.sol]:
```suggestion
    /// @notice Registers an {IInverterBeacon_v1} as a linked beacon that is not a module.
```

File: src/external/governance/interfaces/IGovernor_v1.sol [hunk 3]
```diff
@@ -267,6 +271,11 @@ interface IGovernor_v1 {
...
+    /// @dev	Can only be accessed by either the `COMMUNITY_MULTISIG_ROLE` or the `TEAM_MULTISIG_ROLE`.
+    /// @param  beacon The {IInverterBeacon_v1} that will be registered.
+    function registerNonModuleBeacons(IInverterBeacon_v1 beacon) external;
```
Comment (marvinkruse) [inline src/external/governance/interfaces/IGovernor_v1.sol]: Would use the singular version, as we just add one?
```suggestion
    function registerNonModuleBeacon(IInverterBeacon_v1 beacon) external;
```

File: src/external/governance/Governor_v1.sol [hunk 4]
```diff
@@ -333,17 +336,25 @@ contract Governor_v1 is
...
         IInverterBeacon_v1 beacon
-    ) external onlyCommunityOrTeamMultisig accessibleBeacon(address(beacon)) {
-        linkedBeacons.push(beacon);
+    ) external onlyCommunityOrTeamMultisig {
+        _addBeaconToLinkedBeacons(beacon);
         moduleFactory.registerMetadata(metadata, beacon);
     }
 
+    /// @inheritdoc IGovernor_v1
+    function registerNonModuleBeacons(IInverterBeacon_v1 beacon)
```
Comment (marvinkruse) [inline src/external/governance/Governor_v1.sol]: Singular as well, as noted in the interface already.
```suggestion
    function registerNonModuleBeacon(IInverterBeacon_v1 beacon)
```

**Review by marvinkruse — CHANGES_REQUESTED** [review]
Looks good besides the small notes, good stuff!

File: src/external/governance/Governor_v1.sol (hunk 1 above)
Comment (marvinkruse) [reply src/external/governance/Governor_v1.sol]: Resolved in my commit.

File: src/external/governance/interfaces/IGovernor_v1.sol (hunk 2 above)
Comment (marvinkruse) [reply src/external/governance/interfaces/IGovernor_v1.sol]: Resolved in my commit.

File: src/external/governance/interfaces/IGovernor_v1.sol (hunk 3 above)
Comment (marvinkruse) [reply src/external/governance/interfaces/IGovernor_v1.sol]: Resolved in my commit.

File: src/external/governance/Governor_v1.sol (hunk 4 above)
Comment (marvinkruse) [reply src/external/governance/Governor_v1.sol]: Resolved in my commit.

**Comment by marvinkruse** [comment]
Resolved my raised comments myself, as I thought they are probably uncritical and you wouldn't care too much - if that is not the case feel free to revert them 🙏

If you think it's fine, then feel free to merge!

**Review by marvinkruse — APPROVED** [review]

**Pull request merged**